### PR TITLE
UI improvements

### PR DESCRIPTION
**Pull request by emanuelschuetze**
Implemented many nice ui improvements (discussed with our designer Jörn).

- item list: reorder buttons
- motion list:
  * move hover menu for edit/delete below submitters
  * limit submitters to one
- motion detail:
  * show ol flush left
  * use 2 or 3 columns in meta box
  * use small buttons for line numbering controls
  * group line number controls in grey tool bar.
  * Show motion comments div only if user is allowed.
- motion config:
  * Deativate comments by default.
  * Deactivate recommendations for motions by default.
- item detail: Show needed buttons only.
- assignment detail: Improve ballot tabs.
- general:
  * move pdf button to right
  * move 'back to overview' button to left
  * update label vor countdown option

### CODE REVIEW

File: openslides/agenda/static/js/agenda/site.js [hunk 1]
```diff
@@ -403,11 +415,11 @@ angular.module('OpenSlidesApp.agenda.site', ['OpenSlidesApp.agenda'])
...
         };
 
-        // check if user is allowed to see 'add me' / 'remove me' button
+        // Return true if the requested user is allowed to do a specific action
+        // and see the corresponding button (e.g. 'add me' or 'remove me').
         $scope.isAllowed = function (action) {
             var nextUsers = [];
-            var nextSpeakers = $filter('filter')($scope.speakers, {'begin_time': null});
-            angular.forEach(nextSpeakers, function (speaker) {
+            angular.forEach($scope.nextSpeakers, function (speaker) {
```
Comment (normanjaeckel) [inline openslides/agenda/static/js/agenda/site.js]: Proposal: Use a switch-case-block.

File: openslides/core/config_variables.py [hunk 2]
```diff
@@ -153,7 +153,7 @@ def get_config_variables():
         name='projector_default_countdown',
         default_value=60,
         input_type='integer',
-        label='Default countdown',
+        label='Predefined seconds of new countdowns',
```
Comment (normanjaeckel) [inline openslides/core/config_variables.py]: Is the translation on client side missing?

File: openslides/motions/config_variables.py [hunk 3]
```diff
@@ -161,7 +161,7 @@ def get_config_variables():
 
     yield ConfigVariable(
         name='motions_comments',
-        default_value=[{'name': 'Comment', 'public': True}],
```
Comment (normanjaeckel) [inline openslides/motions/config_variables.py]: Hm. I think one public comment field by default would be very nice for newcomers, right?

File: openslides/motions/static/js/motions/site.js [hunk 4]
```diff
@@ -1374,6 +1375,18 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions', 'OpenSlid
                     $scope.showVersion(motion.active_version);
                 });
         };
+        // check if user is allowed to see at least one comment field
+        $scope.isAllowedToSeeCommentField = function () {
+            var isAllowed = false;
+            if ($scope.commentsFields.length > 0) {
+                angular.forEach($scope.commentsFields, function(field) {
```
Comment (normanjaeckel) [inline openslides/motions/static/js/motions/site.js]: You can use a lodash function the breaks after the first truthy return value. https://lodash.com/docs/4.16.2#find

File: openslides/agenda/static/js/agenda/site.js (hunk 1 above)
Comment (emanuelschuetze) [inline openslides/agenda/static/js/agenda/site.js]: Done

File: openslides/core/config_variables.py (hunk 2 above)
Comment (emanuelschuetze) [inline openslides/core/config_variables.py]: Done

File: openslides/motions/static/js/motions/site.js (hunk 4 above)
Comment (emanuelschuetze) [inline openslides/motions/static/js/motions/site.js]: Done

File: openslides/motions/config_variables.py (hunk 3 above)
Comment (emanuelschuetze) [inline openslides/motions/config_variables.py]: No, I want to use a simple default setup. Motion comments is an extended feature (like supporters, amendments etc.) which should be activate if really needed.

**Comment by emanuelschuetze** [comment]
All done. Ready for review.

File: openslides/motions/static/js/motions/site.js [hunk 5]
```diff
@@ -1379,14 +1379,14 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions', 'OpenSlid
         $scope.isAllowedToSeeCommentField = function () {
             var isAllowed = false;
             if ($scope.commentsFields.length > 0) {
-                angular.forEach($scope.commentsFields, function(field) {
+                _.find($scope.commentsFields, function(field) {
```
Comment (normanjaeckel) [inline openslides/motions/static/js/motions/site.js]: This is not what I meant: `_.find` returns the first element where the predicate function returns a truthy value for. If nothing is found, it returns `undefined`. So your predicate function should return something to work properly and to be an improvement compared to the `angular.forEach`:
```javascript
$scope.isAllowedToSeeCommentField = function () {
    var isAllowed;
    if ($scope.commentsFields.length > 0) {
        isAllowed = operator.hasPerms('motions.can_see_and_manage_comments') || _.find(
            $scope.commentsFields,
            function (field) {
                return field.public
            }
        );
    }
    return Boolean(isAllowed);
};

**Comment by emanuelschuetze** [comment]
@normanjaeckel Ready for review.

**Comment by normanjaeckel** [comment]
Ok, rebase and merge.

**Comment by emanuelschuetze** [comment]
Rebased.

**Pull request merged**